man/man8/zpool-status.8

```diff
@@ -159,6 +171,175 @@ rpool       14.6G  54.9G      4     55   250K  2.69M
 ----------  -----  -----  -----  -----  -----  -----  ----
 .Ed
 .
+.Ss Example 2 : No Display the status output in JSON format
+.Nm zpool Cm status No can output in JSON format if
+.Fl j
+is specified.
+.Fl c
+can be used to run a script on each VDEV.
+.Bd -literal -compact -offset Ds
+.No # Nm zpool Cm status Fl j Fl c Pa vendor , Ns Pa model , Ns Pa size | Nm jq
+{
+  "output_version": {
+    "command": "zpool status",
+    "vers_major": 0,
+    "vers_minor": 1
+  },
+  "pools": {
+    "tank": {
+      "name": "tank",
+      "state": "ONLINE",
+      "guid": "3920273586464696295",
+      "txg": "16597",
+      "spa_version": "5000",
+      "zpl_version": "5",
+      "status": "OK",
+      "vdevs": {
+        "tank": {
+          "name": "tank",
+          "alloc_space": "62.6G",
+          "total_space": "15.0T",
+          "def_space": "11.3T",
+          "read_errors": "0",
+          "write_errors": "0",
+          "checksum_errors": "0",
+          "vdevs": {
+            "raidz1-0": {
+              "name": "raidz1-0",
+              "vdev_type": "raidz",
+              "guid": "763132626387621737",
+              "state": "HEALTHY",
```

Comment:
`zpool list -j | jq` and `zpool status` show the state as "ONLINE", but `zpool status -j | jq` shows the same state as `HEALTHY`. Is this intentional?